### PR TITLE
feat: toggle theme mode with ctrl+shift+L

### PR DESCRIPTION
**Pull request by MayurSMahajan**
### Feature Preview

https://github.com/AppFlowy-IO/AppFlowy/assets/47064215/7beee06b-9a2c-45a1-9d46-5a2ad22f52dc

Solves: #2304 
---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/workspace/application/appearance.dart [hunk 1]
```diff
@@ -49,6 +49,16 @@ class AppearanceSettingsCubit extends Cubit<AppearanceSettingsState> {
...
   }
 
+  /// Toggle the theme mode
+  void toggleThemeMode() {
+    final currentThemeMode = state.themeMode;
+    if (currentThemeMode == ThemeMode.light) {
+      setThemeMode(ThemeMode.dark);
+    } else {
+      setThemeMode(ThemeMode.light);
+    }
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/workspace/application/appearance.dart:59]:
```suggestion
    setThemeMode(currentThemeMode == ThemeMode.light ? ThemeMode.dark : ThemeMode.light);
```

File: frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart [hunk 2]
```diff
@@ -25,6 +26,21 @@ class HomeHotKeys extends StatelessWidget {
             .add(const HomeSettingEvent.collapseMenu());
       },
     );
+
+    final HotKey hotKeyForThemeModeChange = HotKey(
+      KeyCode.keyL,
+      modifiers: [
+        Platform.isMacOS ? KeyModifier.meta : KeyModifier.control,
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: I think you need to add a `!kIsWeb` check, to not break on Web.

Comment (MayurSMahajan) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: How can this break on web?

Comment (Xazin) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: You can't use Platform checks on web

Comment (Xazin) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: Just double-checked, and yes `Platform._operatingSystem` is unsupported on web.

Comment (MayurSMahajan) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: I did not see something that prevents these hotkeys to be used on the web. There is also one hotkey for `Ctrl/Meta + \` to hide/show the left menu bar. There also we don't check for web. 
I guess we can not register both of these hotkeys on web, if they do break the web. Or should I not check the platform and just assign Ctrl key as modifier? What do you think is better? @Xazin @LucasXu0

Comment (Xazin) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart:33]: I guess this is my oversight, I forgot we don't need to support web platform in the Flutter code on the main project.

**Comment by LucasXu0** [comment]
@MayurSMahajan LGTM. Please merge the main branch and resolve the conflict.

**Pull request closed without merging**